README.md

```diff
@@ -92,83 +99,85 @@ The script `hack/test/local_test.sh` can be used to automate local testing by cr
 
 ### Certman Operator Configuration
 
-A [ConfigMap](https://docs.openshift.com/container-platform/3.11/dev_guide/configmaps.html) is used to store certman operator configuration. The ConfigMap contains one value, `default_notification_email_address`, the email address to which Let's Encrypt certificate expiry notifications should be sent.
+A [ConfigMap](https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-configmaps.html) is used to store certman operator configuration. The ConfigMap contains one value, `default_notification_email_address`, the email address to which Let's Encrypt certificate expiry notifications should be sent.
 
-```
+```shell
 oc create configmap certman-operator \
     --from-literal=default_notification_email_address=foo@bar.com
 ```
 
 ### Certman Operator Secrets
 
-A [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) is used to store the Let's Encrypt account url and keys.
+There are two [secrets](https://kubernetes.io/docs/concepts/configuration/secret/) required for certman-operator to function.
+
+1. `lets-encrypt-account` - This secret is used to store the Let's Encrypt account url and keys.
 
+```shell
+ oc create secret generic lets-encrypt-account \
+    --from-file=private-key=private-key.pem \
+    --from-file=account-url=account.txt
 ```
- oc create secret generic lets-encrypt-account-staging \
-    --from-file=private-key=production-private-key.pem \
-    --from-file=account-url=production-account.txt
- oc create secret generic lets-encrypt-account-production \
-    --from-file=private-key=staging-private-key.pem \
-    --from-file=account-url=staging-account.txt
+
+2. `aws` or `gcp` - Based on which platform is being used (AWS or GCP), this is another secret which contains the cloud platform credentials.
+
+```bash
+oc -n certman-operator create secret generic aws --from-literal=aws_access_key_id=XXX 
+--from-literal=aws_secret_access_key=YYYY 
 ```
 
+*NOTE: For testing purposes, both the secrets can be found on the Hive shard of the staging cluster.*
```

Comment:
Similarly, whilst this is true for the `aws` / `gcp` secret that it is found on the Hive, perhaps more clarification needed here.

Reply:
Thanks for the feedback Matt. I have added the required changes. Let me know if this seems ok.

Reply:
Thanks for the clarification there. :+1: